libs/compute_api/src/spec.rs

```diff
@@ -252,6 +262,17 @@ pub enum ComputeMode {
     Replica,
 }
 
+/// Log level for audit logging
+/// Disabled, log, hipaa
+/// Default is Disabled
+#[derive(Clone, Debug, Default, Eq, PartialEq, Deserialize, Serialize)]
+pub enum ComputeAudit {
+    #[default]
+    Disabled,
+    Log,
+    Hipaa,
```

Comment:
Again NIT about naming, but
- Log doesn't tell me anything, I was thinking about Base or something
- Hipaa is too specific, if we later need to enable it for non-HIPAA customer, that'd be awkward, so something like Full or Extended is better, imo

Yet, up to you

Reply:
I'll leave it as is, because that's what we already have in cloud code.

Log - my idea was that this flag will enable pgaudit logging into regular postgres log stream.

hipaa - we may change it to something more generic, like Compliance, later.
Right now the infra (`audit_logging_endpoint` collector and Loki) is also Hipaa specific, and I am not sure if it will be reused by other methods or we will just add another field to this spec with slightly different logic.